src/test/SDL_test_memory.c

```diff
@@ -280,7 +296,20 @@ void SDLTest_TrackAllocations(void)
     if (s_previous_allocations != 0) {
         SDL_Log("SDLTest_TrackAllocations(): There are %d previous allocations, disabling free() validation", s_previous_allocations);
     }
-#ifdef SDL_PLATFORM_WIN32
+#ifdef HAVE_LIBUNWIND_H
+    do {
+        /* Don't use SDL_GetHint: SDL_malloc is off limits. */
+        const char *env_early_procname = SDL_getenv("SDLTEST_EARLY_PROCNAME");
```

Comment:
Do you have a better suggestion?
When `unw_get_proc_name_by_ip` is not available, this variable controls whether we do address -> procname conversion during `SDL_malloc`. If it's unset, no conversion is done and all you get is addresses.

Reply:
`SDL_TRACKMEM_SYMBOL_NAMES`?